### PR TITLE
Google cloud logging Orbit 

### PR DESCRIPTION
**Pull request by sumedhassk**

### CODE REVIEW

File: google-cloud-logging/1.2.1.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<!--
+ ~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thusithathilina) [inline google-cloud-logging/1.2.1.wso2v1/pom.xml]: Year should be 2017

File: google-cloud-logging/1.2.1.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                        <Import-Package>
+                            *;resolution:=optional
+                        </Import-Package>
+                        <Export-Package>
+                            com.google.cloud.logging.*;version="${google-cloud-logging.version}",
+                            com.google.api.gax.paging.*;version="${google-api-extensions-for-java.version}"
+                        </Export-Package>
+                        <Embed-Dependency>*;scope=compile|runtime;inline=false</Embed-Dependency>
+                        <Embed-Transitive>true</Embed-Transitive>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```
Comment (thusithathilina) [inline google-cloud-logging/1.2.1.wso2v1/pom.xml]: Is there a possibility to get rid of Dynamic imports?

**Comment by sumedhassk** [comment]
@thusithathilina Could you please approve the PR.?

File: google-cloud-logging/1.2.1.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Import-Package>
+                        </Import-Package>
+                        <Export-Package>
+                            com.google.cloud.logging.*;version="${google-cloud-logging.version}",
+                            com.google.api.gax.paging.*;version="${google-api-extensions-for-java.version}"
+                        </Export-Package>
+                        <Embed-Dependency>*;scope=compile|runtime;inline=false</Embed-Dependency>
+                        <Embed-Transitive>true</Embed-Transitive>
```
Comment (daneshk) [inline google-cloud-logging/1.2.1.wso2v1/pom.xml]: @sumedhassk, why are we embedding all transitive dependencies in to the bundle?

Comment (sumedhassk) [reply google-cloud-logging/1.2.1.wso2v1/pom.xml]: @daneshk Will check if we can remove this.

**Comment by sumedhassk** [comment]
Closing this PR since there is a new version of the Google cloud logging library. Will send a new PR for that.

**Pull request closed without merging**